contracts/Core/interface/IStateManager.sol

```diff
@@ -3,7 +3,6 @@ pragma solidity ^0.8.0;
 
 
 interface IStateManager {
-    // for testing only. diable in prod
     function setEpoch (uint256 epoch) external;
```

Comment:
These functions were removed above. Interfaces should also be removed

https://github.com/razor-network/contracts/pull/99/files#diff-d713f862919618dee72d779b73d4a918a515ddcddba767aaafe6a17ab1e77d37L13